x/feemarket/keeper/abci.go

```diff
@@ -19,14 +17,6 @@ func (k *Keeper) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
 	}
 
 	k.SetBaseFee(ctx, baseFee)
-
-	// Store current base fee in event
-	ctx.EventManager().EmitEvents(sdk.Events{
-		sdk.NewEvent(
-			types.EventTypeFeeMarket,
-			sdk.NewAttribute(types.AttributeKeyBaseFee, baseFee.String()),
-		),
-	})
```

Comment:
I think we should still emit this event for clients that would like to track the base fee

Reply:
yes I also realized we need to keep retrocompatible, I will add it back